tests/tensor/test_var.py

```diff
@@ -245,8 +246,13 @@ def test__getitem__newaxis(x, indices, new_order):
 
 def test_fixed_shape_variable_basic():
     x = TensorVariable(TensorType("int64", shape=(4,)), None)
-    assert isinstance(x.shape, Constant)
-    assert np.array_equal(x.shape.data, (4,))
+    assert x.type.shape == (4,)
```

Comment:
In the original issue you said that you should not get constants in data, can it be verified explicitly? So far it is not obvious for me